topic_folders/fundraising/collaborating-on-grants.md

```diff
@@ -1,22 +1,22 @@
 # A Guide for Collaborating with The Carpentries on Grants
 
 ## Statement to Interested Collaborators
-This guide was developed for potential collaborators interested in involving The Carpentries in a grant proposal. We value all contributions to our mission and are excited to collaborate with individuals and organisations interested in mission-aligned activities (i.e. Carpentries workshops, instructor training, curriculum development). We offer two options for collaborating with The Carpentries on grant proposals:
+This guide was developed for potential collaborators interested in involving The Carpentries in a grant proposal. We value all contributions to our mission and are excited to collaborate with individuals and organisations interested in mission-aligned activities (i.e., Carpentries workshops, instructor training, curriculum development). We offer two options for collaborating with The Carpentries on grant proposals:
 
-1. **Request a letter of support**. If you are planning to budget Carpentries programming (i.e. workshops, membership, instructor training seats) into your grant proposal and would like to request a letter of support, please [complete this form](https://carpentries.typeform.com/to/oK25vSPE). Pricing for Carpentries programming is located on our [membership page](https://carpentries.org/membership/). All requests for letters of support require **five business days** to be fulfilled.
+1. **Request a letter of support**. If you are planning to budget Carpentries programming (i.e., workshops, membership, instructor training seats) into your grant proposal and would like to request a letter of support, please [complete this form](https://carpentries.typeform.com/to/oK25vSPE). Pricing for Carpentries programming is on our [membership page](https://carpentries.org/membership/). All requests for letters of support require **five business days** to be fulfilled.
 
-2. **Request to meet one of our program leads**. If you have an idea that will support one of the Carpentries' existing priorities and would like to co-write a grant proposal, please schedule a meeting with the corresponding program lead, using the links below. [Minimum requirements](https://docs.google.com/document/d/1-08lXysNEDc0lka0UB6cwToTq7XKi79lgZVMQhOcGZo/edit#heading=h.ucwcxk7d18gv) for co-writing a grant proposal with The Carpentries are provided below.
+2. **Request to meet one of our program leads**. If you have an idea supporting one of the Carpentries' priorities (i.e., capital campaign, community health, process improvement) and would like to co-write a grant proposal, please schedule a meeting with the corresponding program lead using the links below. [Minimum requirements](https://docs.google.com/document/d/1-08lXysNEDc0lka0UB6cwToTq7XKi79lgZVMQhOcGZo/edit#heading=h.ucwcxk7d18gv) for co-writing a grant proposal with The Carpentries are provided below.
```

Comment:
@maneesha the priorities were added in the parentheses.

Reply:
@kariljordan Do we have a description we can point to for the priority areas? To provide more context on what is meant by capital campaign, community health.....

Reply:
- Capital Campaign: Concerted effort to generate unrestricted revenue.
- Community Health: Concerted effort to recognize, empower, and motivate community members.
- Process Improvement: Concerted effort to improve all operational and programmatic workflows; improve and standardise onboarding and offboarding processes for all community roles and segments; and implement automation to reduce administrative time on routine tasks.